projects/docs-tools/src/modules/demo-page/demo-page-type-definitions.component.ts

```diff
@@ -37,6 +40,20 @@ export class SkyDocsDemoPageTypeDefinitionsComponent implements OnInit {
       this.types = this.typeDefinitionService.getTypeDefinitions(
         this.moduleSourceCodePath
       );
+      if (this.additionalSourceCodePaths && this.additionalSourceCodePaths.length > 0) {
+        for (let path of this.additionalSourceCodePaths) {
+          const additionalTypes = this.typeDefinitionService.getTypeDefinitions(
+            path
+          );
+
+          for (let key of Object.keys(this.types)) {
+            /* Sanity check */
+            if (this.types[key] instanceof Array) {
+              this.types[key] = this.types[key].concat(additionalTypes[key]);
```

Comment:
Instead of concatinating the types, you could add an optional argument to the `getTypeDefinitions` method, so that the types are collected correctly, and in alphabetical order.
```
public getTypeDefinitions(sourceCodePath: string, additionalSourceCodePaths?: []): SkyDocsTypeDefinitions
```